CHANGELOG

```diff
@@ -17,18 +17,26 @@ PR - Pull Request (on GitHub), i.e. integration of a new feature or bugfix
 ----                                OpenMS 3.1.0     (released: under development)    ----
 ------------------------------------------------------------------------------------------
 
+We are excited to share an experimental update with our community, mainly targeted at platforms that depend on Conda packages.
+Please note that this is a partial and unannounced release, focused on delivering novel features and major changes that we're actively testing and refining.
+While these features are still in an experimental phase, we encourage adventurous users and platform integrators to explore and provide feedback.
+
 - Major change: removed the distinction between TOPP tool and util and the TOPP and UTIL build targets were merged to a single TOPP target. All utils are considered tools now and categorized as "Utilities".
 - Added SageAdapter. Support for standard identification tasks with sage (https://github.com/lazear/sage).
-- require some advanced instruction sets for x64 CPUs: SSE3 (g++/clang) or AVX (MSVC); and NEON for ARM64 CPUs (#6978)
+- Require some advanced instruction sets for x64 CPUs: SSE3 (g++/clang) or AVX (MSVC); and NEON for ARM64 CPUs (#6978)
   and report them via the OpenMSInfo tool (#7022)
 - Base64 encoding/decoding using the SIMDe library (#6978)
-- fix a crash when loading mzML data with multiple threads which contains non-MS spectra, e.g. 'electromagnetic radiation spectrum' (#7011)
+- Fix a crash when loading mzML data with multiple threads which contains non-MS spectra, e.g. 'electromagnetic radiation spectrum' (#7011)
 - InternalCalibration: improve visualization of calibration plots (#7064)
-- report reading/writing throughput (MiB/sec) when loading/storing mzML (#7035)
-- restore TOPPAS tutorial (#7076)
-- restore developer quick guide in Doxygen docu - see https://openms.de/current_doxygen/html/index.html (#7109)
+- Report reading/writing throughput (MiB/sec) when loading/storing mzML (#7035)
+- Restore TOPPAS tutorial (#7076)
+- Add ability to create decoy features in FeatureFinderIdentification
+- Filter features in ProteomicsLFQ according to feature p-value (additional parameters feature_with(out)_id_min_score, transfer_ids aka MBR is automatically active if targeted_only = false)
+- various low impact UBSan fixes
```

Comment:
move miscellaneous changes to their own section, so as to not distract/overwhelm the reader :-D